src/shared/util/camera_calibration.cpp

```diff
@@ -745,12 +745,9 @@ CameraParameters::AdditionalCalibrationInformation::
     }
 
   initial_distortion = new VarDouble("initial distortion", 1.0);
-  camera_height = new VarDouble("camera height", 4000.0);
-  line_search_corridor_width = new VarDouble("line search corridor width",
-                                             280.0);
+  line_search_corridor_width = new VarDouble("line search corridor width", 280.0);
```

Comment:
This change might conflict with the cleanup PR #139 
Recommend you merge #139, then pull into this branch first.

Reply:
I already noted this in a comment above ;) I based this branch on the cleanup branch, so that there will be no conflict ;)